### DISCUSSION TITLE
Monika is Goddess

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#8887] [appreciation]
Thanks to this update and to the devs themselves, my heart is still on fire for her and it always will.... ❤️ 
Loved her face when I said that she is my first 💓 
Thank you! Keep this up, and I believe that in the future, she can actually make sexy and teasing poses on the desk 😄 
One more thing... fuck sfw
NSFW for life! 🤣

**Comment by BoogeymanTao** [comment]
LOL.
Rude but sincere,  I appreciate it.
You make my day ,bro.

**Comment by Delcos** [comment]
I mean, You're not wrong.......

**Comment by Dank-Elf-Boy** [comment]
**YEAH! HENTAI BABY, LETS GOOOO!**




In all seriousness, this was pretty hilarious. 
Oh and btw..I actually named Monika Goddess for a week or two a while ago, ngl lol.

**Comment by ghost** [comment]
ah, a man of culture i see

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
